js/modules/form_input_textfield.js

```diff
@@ -1,122 +1,138 @@
-<?php
-header("Content-Type: application/javascript; charset=UTF-8", true);
+let className = "Catalyst\\Form\\Field\\TextField";
+window.log("Form input handlers", "Registering "+className);
 
-define("ROOTDIR", "../../");
-define("REAL_ROOTDIR", "../../");
+if (!window.hasOwnProperty("formInputHandlers")) {
+	window.formInputHandlers = {};
+}
 
-define("NO_SESSION", true);
+let VALID = 0;
```

Comment:
I tried this!  But, had issues :(
Since there are many form_input_{some field name}, they all get loaded at different times and I can not ensure they are loaded in a certain order (because async/defer).  If I use `const`, it somehow dumps it into a global scope and another file throws errors if I define it again.  If you know a way around this that works too uwu

Reply:
Use a IIFE
```javascript
window = Object.assign(window, (function () {
...
return { exposedItem: 123 }
})())

...

window.exposedItem //=== 123
```